Jenkinsfile

```diff
@@ -122,78 +275,80 @@ try {
   }
 
```

Comment:
SonarQube needs to be able to get frontend and backend reports.
Could you add a quality gate step before the package step ?

Reply:
currently, sonar analyses only backend report
If we want to improve scope of analyze, this will be done in another PR

Reply:
eg : https://github.com/centreon/centreon/commit/4ae1b718735e95849f53be942b094a594ec5e5d9#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R118

Reply:
then, this will be manage by the merge with your changes